students/JasneetChandok/lesson05/assignment/tests/database.py

```diff
@@ -0,0 +1,77 @@
+import csv
+import os.path
+from pymongo import MongoClient
+
+mongo = MongoClient("localhost:27017")
+
+db = mongo["norton"]
+
+
+def import_data(data_dir, *files):
+    added = [0, 0, 0]
+    errors = [0, 0, 0]
+    reference = 0
+    for filepath in files:
+        collection_name = filepath.split(".")[0]
+        with open(os.path.join(data_dir, filepath)) as file:
+            reader = csv.reader(file, delimiter=",")
+            header = False
+
+            for row in reader:
+                try:
+                    if not header:
+                        header = [h.strip("\ufeff") for h in row]
+                    else:
+                        data = {header[i]:v for i, v in enumerate(row)}
+                        cursor = db[collection_name]
+                        cursor.insert_one(data)
+                        added[reference] +=1
+                except Exception:
+                    errors[reference] += 1
+            reference += 1
+    return tuple(added), tuple(errors)
+
+
+def show_available_products():
+    col = db["product"]
+    query = {"quantity_available":{"$gt":"0"}}
+    final_dict = {}
+    available_products = iter(col.find(query))
+    for product in available_products:
+        prod_id = product["ï»¿product_id"]

```

Comment:
I know several other students had an issue with this weird character showing up. Are you on Windows? 

Something that the other students tried was to remove that from the string